src/parser.rs

```diff
@@ -74,109 +76,227 @@ pub enum Fragment {
 }
 
 macro_rules! delimited {
-    ($i:expr, $submac:ident!( $($args:tt)* )) => {
-        alt!($i,
-            parens!($submac!($($args)*)) => { |(_, content)| (Delimiter::Parenthesis, content) } |
-            braces!($submac!($($args)*)) => { |(_, content)| (Delimiter::Brace, content) } |
-            brackets!($submac!($($args)*)) => { |(_, content)| (Delimiter::Bracket, content) }
-        )
+    ($content:ident in $cursor:expr) => {
```

Comment:
Looks like this code was copied out of syn -- all of these functions are internal unstable APIs and are subject to change at any time.